lib/node/node.js

```diff
@@ -81,6 +81,11 @@ class Node extends EventEmitter {
     if (config.has('logger'))
       logger = config.obj('logger');
 
+    const plugins = this.config.array('plugins', []);
```

Comment:
Does this mean we add `bindex` by default if indexing is turned on? Does it make a difference that it's in a separate module now?

Reply:
By implicitly enabling `bindex` plugin we can ease the migration. No config files need be changed.